yarn-error.log

```diff
@@ -0,0 +1,53 @@
+Arguments: 
```

Comment:
This file should probably be added to .gitignore instead.

Reply:
Ah, I overlooked that, sorry. Yes, I agree. I'll commit that change momentarily.